### DISCUSSION TITLE
Alerts into splunk?

### DISCUSSION THREAD

**Discussion by dootdootbeepbeep** [Security-Onion-Solutions/securityonion#2933] [Unsupported Versions] [answered]
Hi everyone,
We have a distributed deployment in our lab and are looking into getting the alerts forwarded into splunk. We use splunk as our primary siem so it'd be nice to at least see the alerts pop up in there.
I did a bunch of googling but find lots of writeups from 2013 and so on. I'm not exactly an expert at this stuff so I figured I'd some there and ask.
We are running SO2.somethingsomething with a forwards, manager and search node setup.
Any advice pointing me in the right direction would be appreciated.

**Comment by dootdootbeepbeep** [comment, marked as answer]
I figured it out. Since I don't know how to delete posts I'll just answer it in case anyone has a similar q.
For anyone struggling like I was I followed the instructions here 
https://community.splunk.com/t5/All-Apps-and-Add-ons/How-do-I-configure-a-Splunk-Forwarder-on-Linux/m-p/72078
Installed the forwarder
Installed the suricota TA
and used the suricota logs at /nms/suricota
Gonna assume it works like that with the other alerting systems unless there's another way..

**Reply by lerouxm3** [reply]
suricota or suricata

**Reply by dootdootbeepbeep** [reply]
yes.
